proknow-sdk-test/RoleTest/RoleSummaryTest.cs

```diff
@@ -60,53 +54,15 @@ public async Task GetAsyncTest()
 
             // Verify the returned role
             Assert.AreEqual(name, gottenRoleItem.Name);
+            Assert.AreEqual(description, gottenRoleItem.Description);
             Assert.IsFalse(gottenRoleItem.Permissions.CanCreateApiKeys);
```

Comment:
Should we test for all other permissions here as well? Like you have in line 54 of RolesTest.cs